app/src/main/java/org/fossasia/pslab/communication/sensors/SSD1306.java

```diff
@@ -0,0 +1,470 @@
+        0xAA, 0x55, 0xAA, 0x55, 0xAA,	0x00, 0x00, 0x00, 0xFF, 0x00,	0x10, 0x10, 0x10, 0xFF, 0x00,
+        0x14, 0x14, 0x14, 0xFF, 0x00,	0x10, 0x10, 0xFF, 0x00, 0xFF,	0x10, 0x10, 0xF0, 0x10, 0xF0,
+        0x14, 0x14, 0x14, 0xFC, 0x00,	0x14, 0x14, 0xF7, 0x00, 0xFF,	0x00, 0x00, 0xFF, 0x00, 0xFF,
+        0x14, 0x14, 0xF4, 0x04, 0xFC,	0x14, 0x14, 0x17, 0x10, 0x1F,	0x10, 0x10, 0x1F, 0x10, 0x1F,
+        0x14, 0x14, 0x14, 0x1F, 0x00,	0x10, 0x10, 0x10, 0xF0, 0x00,	0x00, 0x00, 0x00, 0x1F, 0x10,
+        0x10, 0x10, 0x10, 0x1F, 0x10,	0x10, 0x10, 0x10, 0xF0, 0x10,	0x00, 0x00, 0x00, 0xFF, 0x10,
+        0x10, 0x10, 0x10, 0x10, 0x10,	0x10, 0x10, 0x10, 0xFF, 0x10,	0x00, 0x00, 0x00, 0xFF, 0x14,
+        0x00, 0x00, 0xFF, 0x00, 0xFF,	0x00, 0x00, 0x1F, 0x10, 0x17,	0x00, 0x00, 0xFC, 0x04, 0xF4,
+        0x14, 0x14, 0x17, 0x10, 0x17,	0x14, 0x14, 0xF4, 0x04, 0xF4,	0x00, 0x00, 0xFF, 0x00, 0xF7,
+        0x14, 0x14, 0x14, 0x14, 0x14,	0x14, 0x14, 0xF7, 0x00, 0xF7,	0x14, 0x14, 0x14, 0x17, 0x14,
+        0x10, 0x10, 0x1F, 0x10, 0x1F,	0x14, 0x14, 0x14, 0xF4, 0x14,	0x10, 0x10, 0xF0, 0x10, 0xF0,
+        0x00, 0x00, 0x1F, 0x10, 0x1F,	0x00, 0x00, 0x00, 0x1F, 0x14,	0x00, 0x00, 0x00, 0xFC, 0x14,
+        0x00, 0x00, 0xF0, 0x10, 0xF0,	0x10, 0x10, 0xFF, 0x10, 0xFF,	0x14, 0x14, 0x14, 0xFF, 0x14,
+        0x10, 0x10, 0x10, 0x1F, 0x00,	0x00, 0x00, 0x00, 0xF0, 0x10,	0xFF, 0xFF, 0xFF, 0xFF, 0xFF,
+        0xF0, 0xF0, 0xF0, 0xF0, 0xF0,	0xFF, 0xFF, 0xFF, 0x00, 0x00,	0x00, 0x00, 0x00, 0xFF, 0xFF,
+        0x0F, 0x0F, 0x0F, 0x0F, 0x0F,	0x38, 0x44, 0x44, 0x38, 0x44,	0x7C, 0x2A, 0x2A, 0x3E, 0x14,
+        0x7E, 0x02, 0x02, 0x06, 0x06,	0x02, 0x7E, 0x02, 0x7E, 0x02,	0x63, 0x55, 0x49, 0x41, 0x63,
+        0x38, 0x44, 0x44, 0x3C, 0x04,	0x40, 0x7E, 0x20, 0x1E, 0x20,	0x06, 0x02, 0x7E, 0x02, 0x02,
+        0x99, 0xA5, 0xE7, 0xA5, 0x99,	0x1C, 0x2A, 0x49, 0x2A, 0x1C,	0x4C, 0x72, 0x01, 0x72, 0x4C,
+        0x30, 0x4A, 0x4D, 0x4D, 0x30,	0x30, 0x48, 0x78, 0x48, 0x30,	0xBC, 0x62, 0x5A, 0x46, 0x3D,
+        0x3E, 0x49, 0x49, 0x49, 0x00,	0x7E, 0x01, 0x01, 0x01, 0x7E,	0x2A, 0x2A, 0x2A, 0x2A, 0x2A,
+        0x44, 0x44, 0x5F, 0x44, 0x44,	0x40, 0x51, 0x4A, 0x44, 0x40,	0x40, 0x44, 0x4A, 0x51, 0x40,
+        0x00, 0x00, 0xFF, 0x01, 0x03,	0xE0, 0x80, 0xFF, 0x00, 0x00,	0x08, 0x08, 0x6B, 0x6B, 0x08,
+        0x36, 0x12, 0x36, 0x24, 0x36,	0x06, 0x0F, 0x09, 0x0F, 0x06,	0x00, 0x00, 0x18, 0x18, 0x00,
+        0x00, 0x00, 0x10, 0x10, 0x00,	0x30, 0x40, 0xFF, 0x01, 0x01,	0x00, 0x1F, 0x01, 0x01, 0x1E,
+        0x00, 0x19, 0x1D, 0x17, 0x12,	0x00, 0x3C, 0x3C, 0x3C, 0x3C,	0x00, 0x00, 0x00, 0x00, 0x00};  //ascii fonts
+
+    private int[] buff;
+    private I2C i2C;
+
+    public SSD1306(Context context) throws IOException, InterruptedException {
+        UsbManager usbManager= (UsbManager) context.getSystemService(Context.USB_SERVICE);  //obtain an instance of UsbManager
+        ScienceLab scienceLab = new ScienceLab(usbManager);
+        this.i2C = scienceLab.i2C;
+        load("logo");
```

Comment:
port only class SSD1306.
"__main__" function stuff is for functionality. Its done here to to show individual functionality.
It would be taken care in sciencelab.java ( means this sensor functions would be exposed when this add-on would be used with device ).